app/main.py

```diff
@@ -44,9 +40,8 @@ def create_smooth_mobile_app(self) -> str:
 class FullStackDeveloper(FrontendDeveloper, BackendDeveloper):
     def __init__(self, name: str) -> None:
         super().__init__(name)
-
-    skills = ["Python", "SQL", "Django",
-              "JavaScript", "HTML", "CSS", ]
+        self.skills = ["Python", "SQL", "Django",
+                       "JavaScript", "HTML", "CSS"]
```

Comment:
you can remove the __init__ in this class if you fix skills append according to previous comments